src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -119,8 +120,9 @@ export default function LineNumberTooltip({
     }
     setTimeout(() => {
       if (lineNumber === lastHoveredLineNumber.current) {
-        if (features.codeHeatMaps) {
-          dispatch(setBreakpointHitCounts(source.id));
+        if (codeHeatMaps) {
+          dispatch(setBreakpointHitCounts(source.id, () => setCodeHeatMaps(false)));
+          dispatch(runAnalysisOnLine(lineNumber));
```

Comment:
I think this is here for the perf win, if HeatMaps fails, but it re-adds the run analysis message which will slow the backend down all the time. It's probably alright to remove this because if HeatMaps fails we'll kick this off anyways